tests/infrared/16/stf-connectors.yaml.template

```diff
@@ -11,15 +11,29 @@ custom_templates:
         OS::TripleO::Services::ComputeCeilometerAgent: /usr/share/openstack-tripleo-heat-templates/deployment/ceilometer/ceilometer-agent-compute-container-puppet.yaml
         OS::TripleO::Services::MetricsQdr: /usr/share/openstack-tripleo-heat-templates/deployment/metrics/qdr-container-puppet.yaml
         OS::TripleO::Services::Redis: /usr/share/openstack-tripleo-heat-templates/deployment/database/redis-pacemaker-puppet.yaml
+        OS::TripleO::Services::GnocchiApi: /usr/share/openstack-tripleo-heat-templates/deployment/gnocchi/gnocchi-api-container-puppet.yaml
+        OS::TripleO::Services::GnocchiMetricd: /usr/share/openstack-tripleo-heat-templates/deployment/gnocchi/gnocchi-metricd-container-puppet.yaml
+        OS::TripleO::Services::GnocchiStatsd: /usr/share/openstack-tripleo-heat-templates/deployment/gnocchi/gnocchi-statsd-container-puppet.yaml
+        OS::TripleO::Services::AodhApi: /usr/share/openstack-tripleo-heat-templates/deployment/aodh/aodh-api-container-puppet.yaml
+        OS::TripleO::Services::AodhEvaluator: /usr/share/openstack-tripleo-heat-templates/deployment/aodh/aodh-evaluator-container-puppet.yaml
+        OS::TripleO::Services::AodhNotifier: /usr/share/openstack-tripleo-heat-templates/deployment/aodh/aodh-notifier-container-puppet.yaml
+        OS::TripleO::Services::AodhListener: /usr/share/openstack-tripleo-heat-templates/deployment/aodh/aodh-listener-container-puppet.yaml
+        OS::TripleO::Services::PankoApi: /usr/share/openstack-tripleo-heat-templates/deployment/deprecated/panko/panko-api-container-puppet.yaml
 
     parameter_defaults:
-        EventPipelinePublishers: []
-        MetricPipelinePublishers: []
-        CeilometerEnablePanko: false
+        CeilometerEnableGnocchi: true
+        GnocchiArchivePolicy: 'high'
+        GnocchiBackend: 'rbd'
+        GnocchiRbdPoolName: 'metrics'
+
+        EventPipelinePublishers: ['gnocchi://?filter_project=service&archive_policy=low']
+        MetricPipelinePublishers: ['gnocchi://?filter_project=service&archive_policy=low']
         CeilometerQdrPublishEvents: true
         CeilometerQdrPublishMetrics: true
+        CeilometerEnablePanko: false
         ManagePolling: true
         ManagePipeline: true
+        NotificationDriver: 'messagingv2'
```

Comment:
Isn't this the default?

Reply:
It's in master, but not in train (yet)

Reply:
I don't understand this change then. What is it providing?

Reply:
It's basically pulling in the change from
https://review.opendev.org/#/c/715394/